packages/create-novu-app/templates/app-react-email/ts/app/novu/emails/novu-onboarding-email.tsx

```diff
@@ -0,0 +1,194 @@
+import {
+  Body,
+  Button,
+  Column,
+  Container,
+  Head,
+  Html,
+  Img,
+  Preview,
+  Row,
+  Section,
+  Text,
+  Hr,
+  Tailwind,
+  render,
+} from "@react-email/components";
+import * as React from "react";
+import { ControlSchema, EmailComponent, PayloadSchema } from "../workflows";
+
+interface NovuWelcomeEmailProps {
+  steps?: {
+    id: number;
+    Description: React.ReactNode;
+  }[];
+  welcomeHeaderText?: string;
+  belowHeaderText?: string;
+  colorChange?: string;
+  components?: EmailComponent[];
+  userImage: string;
+  teamImage: string;
+  arrowImage: string;
+}
+
+const baseUrl = process.env.VERCEL_URL
+  ? `https://${process.env.VERCEL_URL}`
+  : "";
+
```

Comment:
This doesn't appear to be used. Can we remove?

```suggestion
const baseUrl = process.env.VERCEL_URL
  ? `https://${process.env.VERCEL_URL}`
  : "";

```